### PR TITLE
Fix incorrect matching

### LINKED ISSUE #3: Incorrect matching
## Description 
If an expected VNode has less children than actual, it passes the assert incorrectly.

## Environment
version: `1.0.5`

## Code to reproduce

```typescript
it('should not match' () => { // This fails
    const actual = h('div.classA', [
        h('div.classB_A'),
        h('div.classB_B'),
        h('div.classB_C')
    ]);

    const expected1 = h('div.classA', [
        h('div.classB_A')
    ]);

    const expected2 = h('div.classA', [
        h('div.classC')
    ]);

    const expected3 = h('div.classA', [
        Wildcard(),
        h('div.classC')
    ]);

    assert.throws(() => assertLooksLike(actual, expected1)); // It won't throw any errors
    assert.throws(() => assertLooksLike(actual, expected2)); // It won't throw any errors
    assert.throws(() => assertLooksLike(actual, expected3)); // It won't throw any errors
});
 ```
 
 ## Others
 This may be a bug but a breaking change.

### PR DESCRIPTION
**Pull request by unhappychoice**
close: #3

### CODE REVIEW

File: test/index.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as assert from 'assert';
 import { h } from 'snabbdom/h';
 import { VNode } from 'snabbdom/vnode';
 import { assertLooksLike, looksLike, Wildcard } from '../src/index';
-import Snabbdom from 'snabbdom-pragma';
+import * as Snabbdom from 'snabbdom-pragma';
```
Comment (unhappychoice) [inline test/index.tsx]: Or we need TS compiler option `esModuleInterOp` is enabled.

ref: https://unional.gitbooks.io/typescript/content/pages/06-files-and-projects/compiler-options.html#esmoduleinterop

**Comment by unhappychoice** [comment]
Would you review this? :pray: @jvanbruegge

**Comment by jvanbruegge** [comment]
Thanks for this PR. I pushed a change that updates the dependencies and adds CI, could you rebase onto the current master? Then I will merge this

**Comment by unhappychoice** [comment]
Rebased!

**Comment by jvanbruegge** [comment]
Released as `1.0.6`

**Pull request merged**